### ISSUE TITLE
deeplab sync BN

### ISSUE THREAD

**Issue by tonghe90** [tensorflow/models#4234]
Is the model you provide trained using Sync Batch Norm? As I can't reach the performance of the paper, and it seems BN is the only reason.

**Comment by aquariusjay** [comment]
You should ask this question to Tensorflow team.
You could also try fine-tuning from our provided checkpoint.

**Comment by tonghe90** [comment]
@aquariusjay thanks for your advice. By the way, may I ask about the training details of COCO? like the max iteration, base_lr and batch size if I want training from begining?

**Comment by aquariusjay** [comment]
We use 500K iterations (but 300K iterations should be enough) with the same setting as in PASCAL, i.e., base_lr = 0.007 and batch size = 16. We did not tune the base_lr. You may also want to increase it slightly, e.g., base_lr = 0.01.

**Comment by tonghe90** [comment]
@aquariusjay much thanks. Is the trainaug and coco trained seperately？As I found the iteration in your provided xception_coco_voc_trainaug is 30000.

**Comment by tianzhi0549** [comment]
@aquariusjay Is 16 the total batch size or just batch size for each clone? If the batch size 16 is just for each clone, how many clones do you use (ie. what is the total batch size). Thank you very much.

**Comment by yjl9122** [comment]
500000 iterations??? So many iterations???
